lib/reporter.js

```diff
@@ -194,6 +199,8 @@ function MochaParallelTests(runner, options) {
 
         if (testsFinished === options.testsLength) {
             customRunner.end();
+            // Force exit from async tests
+            process.exit(hasFailedTests ? 1 : 0);
```

Comment:
For consistency with node-0.10 it would be right to return 8 here.
See https://github.com/mmotkina/mocha-parallel-tests/blob/master/test/missing-test/index.js